query-engine/connector-test-kit/src/test/scala/writes/ids/relation_pks/SinglePKRelationFieldSpec.scala

```diff
@@ -14,225 +14,6 @@ import util._
 //
 // 2) Checks basic cursor functionality.
 class SinglePKRelationFieldSpec extends FlatSpec with Matchers with ApiSpecBase {
-  // Mutations in this test:
-  //  create         | root   | checked
-  //  update         | root   | checked
-  //  delete         | root   | not possible (1!:1!)
-  //  upsert         | root   | checked
-  //  updateMany     | root   | unnecessary
-  //  deleteMany     | root   | unnecessary
-  //  nested create  | create | checked
-  //  nested update  | update | checked
-  //  nested connect | -      | not possible (1!:1!)
-  //  nested disconn | -      | not possible (1!:1!)
-  //  nested delete  | -      | not possible (1!:1!)
-  //  nested set     | -      | not possible (1!:1!)
-  //  nested upsert  | -      | not possible (1!:1!)
-  //  nested deleteM | -      | not possible (1!:1!)
-  //  nested updateM | -      | not possible (1!:1!)
-  "Using an ID that is also a 1!:1! single-field relation" should "work" in {
```

Comment:
same as above. Should probably bring it back.

Reply:
There are 1:1! cases in the file covering this.